volume-providers/vpc/messages/messages_en.go

```diff
@@ -12,10 +12,18 @@ package messages
 
 import (
 	util "github.com/IBM/ibmcloud-storage-volume-lib/lib/utils"
+	"github.com/IBM/ibmcloud-storage-volume-lib/lib/utils/reasoncode"
 )
 
 // messagesEn ...
 var messagesEn = map[string]util.Message{
+	string(reasoncode.ErrorRequiredFieldMissing): util.Message{
+		Code:        string(reasoncode.ErrorRequiredFieldMissing),
```

Comment:
I was thinking to let this depend on individual volume provider and restrict only `Type`

Reply:
Typecast will go away